content/en/blog/2023/ambient-performance-savings/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Cut Service Mesh Overhead with Istio Ambient Mesh"
+description: An anaylsis of how cloud resources are utilized in ambient and sidecar service mesh architectures.
+publishdate: 2023-07-14
+attribution: "Greg Hanson - Solo.io"
+keywords: [istio,ambient,waypoint,ztunnel]
+---
+
+Istio Ambient Mesh is the new Istio data plane without requiring sidecars, refer to the [announcement](/blog/2022/introducing-ambient-mesh/) for more details. Istio Ambient Mesh is included in [Istio 1.18](https://github.com/istio/istio/releases/tag/1.18.0) but is still considered to be in [alpha feature status](/docs/releases/feature-stages/#feature-phase-definitions).
+
+Sidecars have been a staple of Istio’s architecture since day one and are responsible for the majority of features available in Istio today. However, sidecars require the injection of an additional container to each Kubernetes pod resource, each of which needs allocated resources from the pod.
+
+Many of you are familiar with the simplified operation brought by the Istio ambient architecture. Let’s explore how ambient mesh cuts the service mesh costs typically associated with sidecars.
```

Comment:
```suggestion
Last year the Istio community announced [ambient mesh](/blog/2022/introducing-ambient-mesh/), a new data plane mode which replaces sidecar proxies with per-namespace proxies.

[Sidecars](/docs/reference/glossary/#sidecar) have been a staple of Istio’s architecture since day one, and are the way that most of Istio's features are provided today.  However, sidecars require the injection of an additional container to each Kubernetes `Pod` resource, and each sidecar container requires resources be allocated to it.

Ambient mesh was released in [alpha](/docs/releases/feature-stages/#feature-phase-definitions) in [Istio 1.18](/news/releases/1.18.x/announcing-1.18/). This makes it easy to install, test, and to preview what it will mean for you when it's production-ready.

Aside from the many other benefits of ambient mesh, the simple fact you no longer have to run as many proxies means that you can expect resource (and thus cost) savings. As you will see, these can be very substantial.
```